reactor-core/src/test/java/reactor/guide/GuideTests.java

```diff
@@ -201,21 +201,20 @@ public void advancedCold() {
 
 	@Test
 	public void advancedHot() {
-		Sink<String> hotSource = Sinks.multicastNoWarmup();
+		Sink<String> hotSource = Sinks.replay(0);
```

Comment:
let's use `Sinks.multicast()`, `replay(0)` being kind of a corner case.
Also change the corresponding snippet in the asciidoc!